demo.js

```diff
@@ -193,54 +193,38 @@ var saveEditedJSON = function () {
 var buildGrid = function () {
 	createGrid();
 	createStyles();
-
+	showWarnings();
 
 	// $('.cell').on('click', function(event) {
 	// 	cellDialog(this, event);
 	// });
 }
 
-$(function () {
-	config = fetchConfig() || config;
-	buildGrid();
+function createAlert(html,$container){
+	if(!$container){
+		$container = $(".alerts-container");
+	}
 
-	$('#open-source-code').on('click', function () {
-		getHTML();
-		getFullSource();
-		highlight();
-		$("#sourceContainer").fadeIn();
-		document.getElementById("defaultOpenTab").click();
-	});
-	//Initialize configuration panel
-	var slider = $("#menu-bar").slideReveal({
-		// width: 100,
-		push: false,
-		position: "right",
-		// speed: 600,
-		trigger: $("#menu-bar .handle"),
-		// autoEscape: false,
-		shown: function (obj) {
-			//obj.find(".handle").html('<span class="glyphicon glyphicon-chevron-right"></span>');
-			obj.find(".handle").html('&#10095;');
-			obj.addClass("left-shadow-overlay");
-		},
-		hidden: function (obj) {
-			//obj.find(".handle").html('<span class="glyphicon glyphicon-chevron-left"></span>');
-			obj.find(".handle").html('&#10094;');
-			obj.removeClass("left-shadow-overlay");
-		}
-	});
+	var $alert=$($("template#alert-template").html());
 
-	$("#menu-bar a").on("click", function () {
-		slider.slideReveal("hide");
-	});
+	$alert.find(".message").html(html);
 
-	window.onpopstate = function (event) {
-		window.location.reload();
-	};
-});
+	$container.append($alert);
+}
 
+function showWarnings(){
+	$(".alerts-container").html("");
 
+	// Auto placement warning
+	for(var i=0;i<config.cells.length;i++){
+		if(config.cells[i].colSpan>config.cols.length){
+			createAlert("Auto-placement of grid cells is not supported on IE11. Hence, Powergrid restricts spanning of cells beyond configured number of columns. See <a target='_blank' href='https://github.com/ZS/powergrid/issues/9'>details</a>");
+			break;
+		}
+	}
+
+	//TODO:Additional warning scenarios can be added here.
```

Comment:
Further conditions can be added here to show additional warnings.

Reply:
it is a good start. But we need to drive this alerts from json config file. They can't be hard-coded in the demo.js

Reply:
Nice point. We can have a separate config file for storing warning texts.